test/load/allocation/README.md

```diff
@@ -156,7 +174,7 @@ by running [`./configure-agones.sh`](configure-agones.sh).
 
 ## Fleet Setting
 
-We used the following fleet configuration:
+We used the following [fleet configuration](./scenario-fleet.yaml):
 
 ```
 apiVersion: "agones.dev/v1"
```

Comment:
nit: we can probably remove the code snippets for `scenario-fleet.yam` and `autoscaler.yaml` to make it a little more manageable since we already have the code links here